### ISSUE TITLE
Failed to import vendor/breakpoint/breakpoint

### ISSUE THREAD

**Issue by ian-bertolacci** [academicpages/academicpages.github.io#356]
When building this repository as it exists, github gives the following error:

> # GitHub Pages failed to build your site.
> Your SCSS file assets/css/main.scss has an error on line 10: File to import not found or unreadable: vendor/breakpoint/breakpoint. Load paths: _sass /hoosegow/.bundle/ruby/2.5.0/gems/jekyll-theme-primer-0.5.4/_sass.

**Comment by amwfarid** [comment]
I can't be too sure of which step resolved my issue, but I'll let you know what I did (Mind you I'm pretty much a noob with Jekyll):

- Rather than forking, I started off by cloning then copying the files to my presently available repo folder.
- I did `bundle install` and `bundle update`.
- When I did `gem list --local`, I couldn't find breakpoint. So, I installed it `gem install breakpoint`.
- For some weird reason, I realized that `vendor` was in `.gitignore`. I removed it, then pushed the files once again.
